styles.css

```diff
@@ -1,116 +1,238 @@
 * {
-    box-sizing: border-box; 
-padding:0; margin: 0}
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
 
 body {
-    background-color: lightyellow;
-    font-family: 'Arial';
+  background-color: lightyellow;
+  font-family: "Arial";
+}
+
+body.dark {
+  background-color: rgb(50, 51, 58);
+  color: hotpink;
+}
+
+.header.dark {
+  background-color: rgb(50, 51, 58);
+}
+
+.navbar.dark {
+  background-color: rgb(50, 51, 58);
 }
 
 .header {
-display: flex;
-align-items: center;
-justify-content: center;
-width: 100%;
-margin: auto;
-background-color: peachpuff;
-border: solid;
-border-width: 5px;
-border-color: hotpink;
-position: sticky;
-top: 0;
-z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin: auto;
+  background-color: peachpuff;
+  border: solid;
+  border-width: 5px;
+  border-color: hotpink;
+  position: sticky;
+  top: 0;
+  z-index: 1;
 }
 
 h1 {
-    color: hotpink;
+  color: hotpink;
 }
 
 .question-card {
-    display: flex;
-    position: relative;
-    margin: 35px auto 0 auto;
-    flex-direction: column;
-    align-items: center;
-    width: 70%;
-    background-color: peachpuff;
-
-border: solid;
-border-color: hotpink;
+  display: flex;
+  position: relative;
+  margin: 35px auto 0 auto;
+  flex-direction: column;
+  align-items: center;
+  width: 70%;
+  background-color: peachpuff;
+
+  border: solid;
+  border-color: hotpink;
+  flex-wrap: wrap;
 }
 .question {
-    padding-top: 40px;
-    color: hotpink;
-    font-size: 20px;
-    text-align: center;
+  text-align: center;
+  margin: 40px;
+  color: hotpink;
+  font-size: 20px;
 }
 
 .answer-button {
-    padding: 5px;
-    margin: 40px auto 0 auto;
-    background-color: peachpuff;
-    border-color: hotpink;
-    border-width: 3px;
-    color: hotpink;
-    font-size: 20px;
+  padding: 5px;
+  margin-bottom: 20px;
+  background-color: peachpuff;
+  border-color: hotpink;
+  border-width: 3px;
+  color: hotpink;
+  font-size: 20px;
 }
 
 .answer-button:hover {
-    background-color: #c394f2
+  background-color: #c394f2;
+}
+
+.hidden {
+  display: none;
 }
 
-.bookmark {
-        position: absolute;
-        top: -25px;
-        right: 10px;
-        width: 50px;
-        height: 50px;
+.answer {
+  color: hotpink;
+  font-size: 20px;
+}
+
+.active {
+  filter: invert(73%) sepia(33%) saturate(1470%) hue-rotate(307deg)
+    brightness(103%) contrast(88%);
+}
+
+.bookmark-button {
+  position: absolute;
+  border: none;
+  background: transparent;
+  top: -25px;
+  right: 10px;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
 }
 
 ul {
```

Comment:
Instead of using a type selector it would be better to use classes in CSS. This way you are ore independent from your markdown with your styling following the [separation of concern](https://en.wikipedia.org/wiki/Separation_of_concerns#HTML,_CSS,_JavaScript) principle.